index.d.ts

```diff
@@ -1,5 +1,13 @@
 declare namespace matcher {
 	interface Options {
+		/**
+		Requires all negated patterns to not match and any normal patterns to match at least once. Otherwise, it will be a no-match condition.
+
+		This option may slow down `matcher.isMatch` with long inputs.
```

Comment:
"long inputs" is too ambitious. Does it means many items or that the individual items have long strings.

Reply:
This was misleading statement altogether, so I removed it - thanks for pointing it out!
Who really needs to know all the tiny details, would examine the code anyway.